### PR TITLE
Adds station art and tests

### LINKED ISSUE #74: Station Art not Showing on Browse
Station art images are not appearing when browsing mopidy-pandora stations.  I'll work on this and submit a pull request when I have it figured out.

### PR DESCRIPTION
**Pull request by abates**
This commit adds station art lookup and an associated test. Fixes #74

### CODE REVIEW

**Comment by abates** [comment]
I'll work on increasing the test coverage.

File: mopidy_pandora/library.py [hunk 1]
```diff
@@ -127,11 +129,39 @@ def get_images(self, uris):
         for uri in uris:
             image_uris = set()
             try:
-                track = self.lookup_pandora_track(uri)
-                if track.is_ad is True:
-                    image_uri = track.image_url
+                image_uri = None
```
Comment (txomon) [inline mopidy_pandora/library.py:132]: I would suggest to split this into a separate function, as the amount of indentation goes up to 7.

File: mopidy_pandora/library.py [hunk 2]
```diff
@@ -145,14 +175,7 @@ def get_images(self, uris):
...
                     )
-                else:
-                    # Lookup
-                    logger.warning(
-                        "No images available for Pandora URIs of type '{}'.".format(
-                            pandora_uri.uri_type
-                        )
-                    )
-                pass
+
```
Comment (txomon) [inline mopidy_pandora/library.py:178]: Given that the amount of exceptions has been reduced, do we need to do the `isinstance` check?

**Review by txomon — COMMENTED** [review]
Just a couple of comments to improve maintenance, just nit-picks

**Comment by jodal** [comment]
@abates This has now been released in 1.1.0. None of the core Mopidy developers lives in a region with Pandora available, so we'd very much like feedback on if it works or not, and even better, a new maintainer for the extension.

**Pull request merged**